build-system/tasks/coverage-map.js

```diff
@@ -43,8 +44,8 @@ async function doTest() {
   log('Testing completed.');
   const jsCoverage = await page.coverage.stopJSCoverage();
   const data = JSON.stringify(jsCoverage);
-  log('Writing to test.json in working directory...');
-  fs.writeFile(coverageJsonName, data, () => {});
+  log(`Writing to ${coverageJsonName} in dist/${coverageJsonName}...`);
+  fs.writeFile(`dist/${coverageJsonName}`, data, () => {});
```

Comment:
nit: I feel like the async/promise version of `fs` is generally preferable to the callback form, and might work better if later we want to `await` it

Reply:
Did you mean by wrapping `fs` inside a promise?

Reply:
I think he means `writeFileAsync`

Reply:
Or `require('fs').promises` (or maybe we use the bluebird promise wrapper, not certain without grepping the code)

Reply:
I was not able to find `writeFileAsync` in the `fs` documentation, but I did find `require('fs').promises`. Therefore, that is what I will use instead.

Reply:
Drive-by comment: We stopped using `bluebird` in #25822. In this instance, since the code is being executed sequentially, I'd recommend using `fs.writeFileSync`.

Reply:
I've updated the import statement to use its async counterpart. Resolving this conversation.

Reply:
@rsimha Could the async counterpart work as well? I believed that it made sense to use async `writeFile` because using mostly `await` inside this task.

Reply:
@rsimha Since it appears the task doesn't need to await/block until the write completes, firing off an async task seems preferable to blocking on the write.

Reply:
I don't have a strong preference between the two, so happy to defer to @rcebulko's suggestion.

Reply:
this is definitely now how you want to use `writeFile`. please use the write in fs-extra or use https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_util_promisify_original the promisify util in node to properly turn the callback style writeFile into a promise

Reply:
Alright. I'll go ahead and resolve this conversation.